CMSgov/eAPD
# bbrooks/4194-Update-PostGres

Resolves # 4194

### Description
Upgrading PostGres version

### Significant changes or possible side effects
Upgrades the version of PostGres, could break the App.

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |

### Steps to manually verify this change

1. Log onto Preview Instance, check PostGres version
2. Check functionality of the site/app
3.

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
